check_api/pom.xml

```diff
@@ -154,6 +154,12 @@
       <artifactId>caffeine</artifactId>
       <version>${caffeine.version}</version>
     </dependency>
+    <!-- Hubspot deps -->
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>2.7.9.5</version>
```

Comment:
Should we use 2.9.9.x or 2.9.10.x here?

Reply:
This is going to be in the same classpath as several other things that are on basepom, so I'd like to keep the version consistent.

Reply:
Ok. We should probably update basepom then (doesn't have to be now)